### ISSUE TITLE
Error on first start - password authentication failed for user "meteor"

### ISSUE THREAD

**Issue by mrn3** [numtel/meteor-pg-server#2]
I am getting issues when I start up for the first time.  I think I installed everything right, and I can get into the postgres database using SQL Workbench.  Do I need to create a meteor user in the database?

```
Matts-MacBook-Pro:meteor-pg-leaderboard Matt$ meteor
[[[[[ ~/Documents/meteor-pg-leaderboard ]]]]]

=> Started proxy.
=> Meteor 1.1.0.3 is available. Update this project with 'meteor update'.
=> Started MongoDB.
[Postgres]  LOG:  database system was shut down at 2015-08-20 06:26:12 GMT

[Postgres]  LOG:  MultiXact member wraparound protections are now enabled

[Postgres]  LOG:  database system is ready to accept connections

=> Started PostgreSQL.
[Postgres]  LOG:  autovacuum launcher started/

W20150820-00:26:19.992(-6)? (STDERR)
W20150820-00:26:19.993(-6)? (STDERR) events.js:72
W20150820-00:26:19.994(-6)? (STDERR)         throw er; // Unhandled 'error' event
W20150820-00:26:19.994(-6)? (STDERR)               ^
W20150820-00:26:19.994(-6)? (STDERR) error: password authentication failed for user "meteor"
W20150820-00:26:19.994(-6)? (STDERR)     at [object Object].Connection.parseE (/Users/Matt/.meteor/packages/numtel_pg/.1.0.0.lrow5g++os+web.browser+web.cordova/npm/node_modules/pg/lib/connection.js:534:11)
W20150820-00:26:19.994(-6)? (STDERR)     at [object Object].Connection.parseMessage (/Users/Matt/.meteor/packages/numtel_pg/.1.0.0.lrow5g++os+web.browser+web.cordova/npm/node_modules/pg/lib/connection.js:361:17)
```

**Comment by numtel** [comment]
If you want to use a user called "meteor" then, yes you do need to create it.

The read me includes the connection string for the default user:

https://github.com/numtel/meteor-pg-server/blob/master/README.md#usage

**Comment by mrn3** [comment]
Okay, that makes sense - I am working on the leaderboard example, and I updated the leaderboard.js to have that connection string.  It would be nice to have a github repo that is the leaderboard example that is all configured with pg-server.  Perhaps I will put that up there once I have it figured out.

**Comment by numtel** [comment]
Yeah I had thought about that. It was not going to be simple with the old version of this package though. I have just released version 1.0.0 which includes an `initialize` setting that can be used to provide a file with SQL commands to run on first installation of the server. The leaderboard example has been updated to include this as well. See the instructions for using an embedded server.

https://github.com/numtel/meteor-pg-leaderboard

Also for mysql:

https://github.com/numtel/meteor-mysql-leaderboard
